projects/core/src/store-finder/occ/store-finder.service.ts

```diff
@@ -3,10 +3,10 @@ import { HttpClient, HttpParams } from '@angular/common/http';
 import { throwError, Observable } from 'rxjs';
 import { catchError } from 'rxjs/operators';
 
-import { SearchConfig } from '../../store-finder/models/search-config';
-import { LongitudeLatitude } from '../../store-finder/models/longitude-latitude';
+import { StoreFinderSearchConfig, LongitudeLatitude } from './../model';
```

Comment:
give explicitly path.
` import { StoreFinderSearchConfig, LongitudeLatitude } from './../model/index';` 
We don't use barrels.

Reply:
I replaced import accordingly to your suggestion.